core/http-auth/src/main/java/software/amazon/awssdk/http/auth/AwsV4HttpSigner.java

```diff
@@ -82,6 +83,47 @@ public interface AwsV4HttpSigner extends HttpSigner<AwsCredentialsIdentity> {
     SignerProperty<Boolean> NORMALIZE_PATH =
         SignerProperty.create(Boolean.class, "NormalizePath");
 
+    /**
+     * The location where auth-related data is inserted, as a result of signing.
+     * The valid choices are "Header" and "QueryString", where "Header" indicates that request
+     * headers are added, and "QueryString" indicates query-parameters are added.
+     * This property defaults to "Header".
+     */
+    SignerProperty<String> AUTH_LOCATION =
```

Comment:
I think this can be an enum instead of String. If folks agree I can make that change.

Reply:
+1 seems good to me